js/src/dapps/registry/accounts/accounts.js

```diff
@@ -0,0 +1,45 @@
+import React, { Component, PropTypes } from 'react';
+import Menu from 'material-ui/Menu';
+import MenuItem from 'material-ui/MenuItem';
+
+// import styles from './lookup.css';
+
+const renderAccount = (active) => (account) => (
```

Comment:
The code is react-only anyway, that function is returning a JSX tree.
imho it's:
- more explicit with inline jsx statement
- could be (smoke-)tested
- could have props validation
- will be easier to extract in case the code grows

This small work that (most probably) will be done in a near future can be done already.